src/data/additionalDocs.js

```diff
@@ -0,0 +1,51 @@
+const additionalDocs = {
+  nerdlet: {
+    methods: [
+      {
+        name: 'nerdlet.setUrlState',
+        description: `
+Updates the current nerdlet's URL state that can be read from
+\`NerdletStateContext.Consumer\`.
```

Comment:
Could we link this to the documentation for `NerdletStateContext` here? Its one step toward solving an issue @cirne raised in #538 about how to use `NerdletStateContext`, granted I think we likely need to do a bit more to fully solve that documentation gap.

Reply:
```suggestion
[\`NerdletStateContext.Consumer\`](/components/nerdlet-state-context).
```

I probably should have provided a suggestion to make it easier on you 😄